frontend/catalyst/compiler.py

```diff
@@ -601,7 +603,9 @@ def get_output_of(self, pipeline) -> Optional[str]:
         Returns
             (Optional[str]): output IR
         """
-        if len(dict(self.options.get_pipelines()).get(pipeline, [])) == 0:
+        if not self.last_compiler_output or not self.last_compiler_output.get_pipeline_output(
+            pipeline
+        ):
```

Comment:
The old line (`self.options.get_pipelines()`) only checks if the given stage is in the MLIR pipeline. If we give it an LLVM pass, it will find nothing. 
The new line (`self.last_compiler_output.get_pipeline_output`) is able to check the given LLVM pass exists or not.